### ISSUE TITLE
Citation placeholder lacks sufficient contrast

### ISSUE THREAD

**Issue by sarahmonster** [WordPress/gutenberg#8738]
It's also super small, which doesn't help matters:

<img width="661" alt="screenshot 2018-08-09 00 12 14" src="https://user-images.githubusercontent.com/376315/43869164-1305f1de-9b69-11e8-8be8-2430bda96418.png">

> The color combination #b6bbc0/#ffffff has a contrast ratio of 1.94, which is not sufficient. At this size, you will need a ratio of at least 4.5.

**Comment by sarahmonster** [comment]
See also: #5838

**Comment by afercia** [comment]
I think also the _quote_ placeholder has contrast issues:

<img width="430" alt="screen shot 2018-10-13 at 15 00 16" src="https://user-images.githubusercontent.com/1682452/46905799-f3112b00-cef8-11e8-9555-9aef54577d1e.png">

**Comment by carolinan** [comment]
The placeholder color has changed several time since this issue was opened, and the current contrast ratio is
6.3:1 which passes WCAG 2.1 level AA.
The problem has been solved and I am closing the issue.
